integreat_cms/cms/constants/roles.py

```diff
@@ -57,6 +60,20 @@
     "view_poi",
 ]
 
+#: The permissions of the limited_page_manager
+OBSERVER_PERMISSIONS = [
+    "add_directory",
+    "change_directory",
+    "view_directory",
+    "change_mediafile",
+    "replace_mediafile",
+    "upload_mediafile",
```

Comment:
Hmm, these permissions do not really fit the role name "observer", but ok, don't see much alternative to it :sweat_smile: